x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/prebuilt_rules_preview.cy.ts

```diff
@@ -337,6 +337,7 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
     type: 'esql',
     language: 'esql',
     query: 'FROM .alerts-security.alerts-default | STATS count = COUNT(@timestamp) BY @timestamp',
+    alert_suppression: undefined,
```

Comment:
Why not pass a mock `alert_suppression` object here and then check that it's rendered like we do for custom query rules? 

Also, I think we need to add alert suppression properties in the test plan under "ESQL" [here](https://github.com/vitaliidm/kibana/blob/eb7db7330e1716d69cc2d867a3021776de583906/x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/installation_and_upgrade.md?plain=1#L200).

Reply:
added in https://github.com/elastic/kibana/pull/180927/commits/8702a4862ca910b459ebbd0a600bc297e22a8628